src/compile_model_manifest.py

```diff
@@ -11,16 +11,28 @@
     model_list = yaml.safe_load(f.read())
     for item in model_list:
         config_url = item['config_url']
+        root_url = '/'.join(config_url.split('/')[:-1])
         response = requests.get(config_url)
         if response.status_code == 200:
             model_config = yaml.safe_load(response.content)
 
             # merge item from models.yaml to model config
             model_config.update(item)
-            model_info = {}
+            model_info = {"root_url": root_url}
             for k in model_config:
+                if k == 'config_url':
+                    model_config[k] = config_url.split('/')[-1]
```

Comment:
If we need to do validation and modification it might be better to do that with a library, eg `voluptuous` so that we can run the whole loaded yaml through a validation schema. Then we don't need to add `if:`  blocks to this loop when we need more validators.

Reply:
I think we won't have many modification like this, a downside of depending on another library and make easy stuff not understandable until you learn that library.

Reply:
That's true. We can evaluate again later if this block grows.